docs/src/Contributing.md

```diff
@@ -70,7 +70,37 @@ in `docs/`.
 When your PR is ready for review, clean up your commit history by squashing
 and make sure your code is current with `ClimateMachine` master by rebasing.
 
-### Squash and rebase
+## Reformatting your code with `.dev/climaformat.jl`
+
+Your PR cannot be merged unless it conforms to the format style specified
+by the formatter implemented in the script `.dev/climaformat.jl`.
+
+To use the script `.dev/climaformat.jl` to reformat every file in
+`ClimateMachine.jl/` run
+
+```
+julia --project .dev/climaformat.jl
```

Comment:
You don't need the `--project`, `climaformat.jl` uses `Pkg.activate` on its own `Project.toml`.

Reply:
in fact it's executable, so you can just do `.dev/climaformat.jl`

Reply:
Ah nice ok.